src/tasks/runstart.ts

```diff
@@ -574,11 +574,15 @@ export const RunStartQuest: Quest = {
             [$item`Apriling band piccolo`, piccoloValue],
           ]),
         ]
-          .filter(
-            ([it]) =>
-              !have(it) && // Remove option if we already have the item
-              !get(`instant_save${it.name.replace(/( \w)/, (_, g) => g.toUpperCase())}`, false), // or if we chose to not acquire it
```

Comment:
Seems like we're just missing the global modifier
We could just do `it.name.replace(/(\s\w)/g, (_, g) => g.toUpperCase()).replace(/\s/g, "")`

We can confirm if this works by testing it in the CLI
```
jsl print($item`Apriling band quad tom`.name.replace(/(\s\w)/g, (_, g) => g.toUpperCase()).replace(/\s/g, ""))
```
to see if it returns `AprilingBandQuadTom`